docs/how-to-docs/cert-manager.md

```diff
@@ -168,30 +202,40 @@ spec:
   - Ingress
   podSelector:
     matchExpressions:
-    - {key: certmanager.k8s.io/acme-http-domain, operator: Exists}
-    - {key: certmanager.k8s.io/acme-http-token, operator: Exists}
+    - {key: acme.cert-manager.io/http-domain, operator: Exists}
+    - {key: acme.cert-manager.io/http-token, operator: Exists}
   ingress:
   - from:
     - namespaceSelector:
         matchLabels:
           name: ingress-external
-    - podSelector:
+      podSelector:
         matchLabels:
           name: ingress
     ports:
     - protocol: TCP
       port: 8089
 ```
 
-#### **As a developer I want to retrieve a certificate for a service behind the vpn, or simple wish to use the DNS validation**
+#### **As a developer I want to retrieve a certificate for a service behind the vpn, or simply wish to use the DNS validation**
 
-As stated above in order to retrieve a certificate for sites which are not externally facing we need to switch to a DNS challenge. Please ensure you have contacted the ACP team before attempting this as the correct permission need to exist to permit the cert-manager to add records to the domain.
+When a site is internal / behind the vpn, in order to handle the challengert you need to switch to using a DNS challenge.
+
+This is done by adding the following to your ingress resource:
+
+- annotation `cert-manager.io/enabled: "true"`
+- label `cert-manager.io/solver: route53`
+
+**Very Important**: in order to successfully switch to a DNS challenge, please ensure you have contacted the ACP team before attempting this for the first time on your sub-domain as the correct permissions need to exist to permit cert-manager to add records to the domain.
 
 ```YAML
-apiVersion: certmanager.k8s.io/v1alpha1
+apiVersion: networking.k8s.io/v1beta1
 kind: Certificate
 metadata:
   name: example-tls
+  labels:
+    # @Note: this label tells the cluster issuer to use the DNS01 Route53 solver instead of the default HTTP01 solver
+    cert-manager.io/solver: route53
```

Comment:
Is there a solution for domains managed via DynDNS yet?

Reply:
Not natively. Possibly if DynDNS (I assume you mean the one from dyn.com) supports https://cert-manager.io/docs/configuration/acme/dns01/rfc2136/ although I found no evidence that it did.